src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] it would be better to use the[ duck patterns](https://github.com/erikras/ducks-modular-redux) as required in the requirements so that you complete the requirements

Reply:
Changes completed rewrite the required code to follow duck patterns